spec/features/login/login_spec.rb

```diff
@@ -0,0 +1,109 @@
+require "rails_helper"
+
+describe "Visitor Login Page" do
+  it "visitor can login if correct infomation is entered " do
+    user = User.create({
+      first_name: "Bob",
+      last_name: "Bobby",
+      email: "bobbobby@bobby.com",
+      password: "password",
+      })
+
+  visit '/'
+  click_on "Login"
+  expect(current_path).to eq("/login")
+
+  fill_in :email, with: user.email
+  fill_in :password, with: user.password
+
+  click_on "Submit"
+
+  expect(current_path).to eq("/profile")
+  expect(page).to have_content("You are now logged in as #{user.first_name}")
+  end
+
+  it "visitor returns to login screen if bad email is entered" do
+    user = User.create({
+      first_name: "Bob",
+      last_name: "Bobby",
+      email: "bobbobby@bobby.com",
+      password: "password",
+      })
+
+      visit '/login'
+
+      fill_in :email, with: "Hello"
+      fill_in :password, with: "password"
+
+      click_on "Submit"
+
+      expect(current_path).to eq("/login")
+      expect(page).to have_content("The credentitals you have entered are invalid")
```

Comment:
Wee typo here with "credentials"

Reply:
actually it looks like this in every test and the controller. Do you mind going through and fixing it?

Reply:
Good catch, should be fixed now.